src/runtime/importhook.cs

```diff
@@ -94,6 +87,10 @@ internal static void Initialize()
             IntPtr dict = Runtime.PyImport_GetModuleDict();
             Runtime.PyDict_SetItemString(dict, "CLR", py_clr_module);
             Runtime.PyDict_SetItemString(dict, "clr", py_clr_module);
+
+            // Add/create the MetaPathLoader
+            SetupNamespaceTracking();
+            PythonEngine.Exec(LoaderCode);
```

Comment:
I think this should rather be an actual module than just a "script" that is executed. Doesn't change much with regard to the C# code, but allows users to access the loader classes directly (e.g. to deactivate the loading for a short while and reactivate it later).